app.js

```diff
@@ -23,11 +23,24 @@ var sectEl = document.getElementById('cookie-list');
 var listElement = document.createElement('ul');
 
 var headerEl = document.createElement('h1');
+var currentHour = 6;
+var currentTime = '';
+var morning = true;
 
 for (var i = 0; i < firstAndPike.avgHours.length; i++) {
   var listLiEl = document.createElement('li');
-
-  listLiEl.textContent = firstAndPike.avgHours[i];
+  if(morning) {
```

Comment:
Is this set of functionality unique to first and pike, or is it shared between all cookie shops?